Assignments/Sorting/SelectionSort.playground/Pages/Table of Contents.xcplaygroundpage/Contents.swift

```diff
@@ -11,3 +11,136 @@ When you ask someone to sort a deck of cards in ascending order by hand, you wil
 ****
 [Start with the first step](@next)
 */
+//
+// Part 1
+//
+
+func selectionSort(originalArray: [Int]) -> [Int] {
+    
+    var newArray = originalArray
+    var startingIndex = 0
+    var otherIndex = 1
+    var arrayLength = newArray.count
+    
+    repeat {
+        repeat {
+            if newArray[startingIndex] > newArray[otherIndex] {
+                swap(&newArray[startingIndex], &newArray[otherIndex])
+                startingIndex = 0
+                otherIndex = 1
+            } else {
+                otherIndex++
+            }
+        } while otherIndex < (arrayLength - 1)
```

Comment:
@diegopetrucci 
If I might suggest something for the sake of consistency:
You could leave the parenthesis out here, like you did in the outer while-loop.

As a side note:
The 'minus' operator has a precedence of 140, whereas the 'less than' operator has a precedence of 130, which means, the 'minus' is computed first.
Precedence means 'priority' in that case.

If you like, you can find more information about operators, precedence and associativity in the documentation:
https://developer.apple.com/library/prerelease/ios/documentation/Swift/Conceptual/Swift_Programming_Language/AdvancedOperators.html
https://developer.apple.com/library/prerelease/ios/documentation/Swift/Reference/Swift_StandardLibrary_Operators/index.html#//apple_ref/doc/uid/TP40016054